From 171d1ef9a056d30378f7f920a2a9931335d05ec7 Mon Sep 17 00:00:00 2001
From: Matthew Rudy Jacobs <matthewrudyjacobs@gmail.com>
Date: Mon, 5 Jan 2009 01:34:26 +0000
Subject: [PATCH] when we run rake rails:freeze:edge
 update the value of the "REVISION" file
 rather than creating a new file "REVISION_ce706..." each time

---
 railties/lib/tasks/framework.rake |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/railties/lib/tasks/framework.rake b/railties/lib/tasks/framework.rake
index 191c936..5dcdace 100644
--- a/railties/lib/tasks/framework.rake
+++ b/railties/lib/tasks/framework.rake
@@ -64,7 +64,10 @@ namespace :rails do
           rm_f goner
         end
 
-        touch "rails/REVISION_#{latest_revision}"
+        puts "Frozen to git revision #{latest_revision}"
+        File.open('rails/REVISION', 'w') do |revision|
+          revision.puts latest_revision
+        end
       end
 
       puts 'Updating current scripts, javascripts, and configuration settings'
-- 
1.5.6.3

